examples/speaker_tasks/diarization/conf/inference/diar_infer_meeting.yaml

```diff
@@ -10,6 +10,9 @@ num_workers: 1
 sample_rate: 16000
 batch_size: 64
 device: null # can specify a specific device, i.e: cuda:1 (default cuda if cuda available, else cpu)
+map_location: null
```

Comment:
I didn;t notice this last time. Is `map_location`  different from `device`?

Reply:
Yes, `map_location` is used to determine if the model weights device. This just exposes the default to ensure when CPU is chosen, we do not load on the GPU.

Reply:
Then I don;t see use of device now. Lets keep only one and remove other

Reply:
Didn't you already add `device` for this reason ? Since inference is only supporting 1 GPU inference, I don't see the need of map_location while we have `device`.

Reply:
@SeanNaren Could you address this question? It will be great to make this clear - why do we need `map_location` for inference

Reply:
Addressed issues, for the config file we rely on `device` which is required for the hydra runner as device needs to be specified beforehand.

Regarding why map location is needed, this is just imitating the PyTorch standard API:

```python
# load the model onto CUDA:1
diarizer = NeuralDiarizer.from_pretrained(model_name='diar_msdd_telephonic', map_location='cuda:1')
diarizer.to(torch.device('cuda'))

annotation = diarizer("an4_diarize_test.wav", max_speakers=4, out_dir='diar/')
```

I think I've addressed this issue though! there is now just one internal variable; device.